### ISSUE TITLE
"Tutorials" Page

### ISSUE THREAD

**Issue by Simranvirdee** [alex0188/MTM6330-Week-5#3]
- The colour contrast with the white text in front of the black background is really nice and creates the text to be more visible
- I'm still not sure what the red box on the side is for
- This page has very minimal information on it but maybe have some text explaining what this page is about and more about the tutorial process
- Overall this page was neatly done
